tests-functional/tests/test_add_account.py

```diff
@@ -0,0 +1,148 @@
+import copy
+import pytest
+from resources.constants import new_account_data_1, new_account_data_2
+
+
+@pytest.mark.rpc
+class TestAddAccount:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.account = backend_new_profile("sender")
+        self.account_data = copy.deepcopy(new_account_data_1)
+
+    def test_add_account_for_valid_key_uid(self):
+        self.account_data["key-uid"] = self.get_keypair_key_uid()
+        add_account_response = self.account.accounts_service.add_account(self.account_data)
+        self.account.verify_json_schema(add_account_response, method="accounts_addAccount")
```

Comment:
why not verify the schema inside `add_account` with `rpc_valid_request`? 🤔 
(same applies to other methods)